indico/htdocs/sass/partials/_core.scss

```diff
@@ -64,7 +64,7 @@
 }
 
 .weak-hidden {
-    display: none;
+    display: none !important;
```

Comment:
FWIW, I think using hidden or weak-hidden in combination with JS widgets is a bad idea. You can't `.show()`, `.hide()`, `.toggle()` those elements while you can do that if you hide them with the good old `style="display: none;""